src/main/groovy/life/qbic/portal/sampletracking/app/PortletController.groovy

```diff
@@ -13,4 +15,5 @@ interface PortletController {
 
     void clearSelection()
 
+    void updateSamples(String sampleIds, Location desiredLocation, Status desiredStatus)
```

Comment:
I would not bundle all sample IDs into one string. Just provide an arbitrary number of String objects. 

```suggestion
    void updateSamples(String... sampleIds, Location desiredLocation, Status desiredStatus, Date date) {
```

Reply:
```suggestion
    void updateSamples(String sampleIds, Location desiredLocation, Status desiredStatus)
```

> I would not bundle all sample IDs into one string. Just provide an arbitrary number of String objects.

Sure, but better pass a List then. Variable arguments as first positional argument is bad practise (https://www.geeksforgeeks.org/variable-arguments-varargs-in-java/)

Reply:
```suggestion
    void updateSamples(List<String> sampleIds, Location desiredLocation, Status desiredStatus)
```